clang/test/Index/Core/scan-deps-by-mod-name.m

```diff
@@ -0,0 +1,34 @@
+// RUN: rm -rf %t.mcp
+// RUN: echo %S > %t.result
+// RUN: c-index-test core --scan-deps-by-mod-name -module-name=ModA %S -- %clang -cc1 -I %S/Inputs/module \
+// RUN:     -fmodules -fmodules-cache-path=%t.mcp -fimplicit-module-maps \
+// RUN:     -o FoE.o -x objective-c >> %t.result
+// RUN: cat %t.result | sed 's/\\/\//g' | FileCheck %s
+
+// Use driver arguments.
+// RUN: rm -rf %t.mcp
+// RUN: echo %S > %t.result
+// RUN: c-index-test core --scan-deps-by-mod-name -module-name=ModA %S -- %clang -c -I %S/Inputs/module \
+// RUN:     -fmodules -fmodules-cache-path=%t.mcp \
```

Comment:
This `RUN` line only differs from the one above in `-fimplicit-module-maps`. AFAIK there's no difference how implicit module maps are handled between dependency scan for a file vs module name. Why do we need to test it here?